src/nupic/bindings/engine_internal.i

```diff
@@ -46,6 +46,8 @@
  * ----------------------------------------------------------------------
 */
 
+#include <nupic/engine/Input.hpp>
+#include <nupic/engine/Link.hpp>
 
```

Comment:
I followed what appears to be a precedent in this file, whereby the `nupic/utils/*` includes were separated by a blank line from the types/ntypes includes.